packages/apps/fortune/exchange-oracle/server/src/modules/assignment/assignment.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class AssignmentStatsDto {
+  assignments_total: number;
+  submissions_sent: number;
+  assignments_completed: number;
+  assignments_rejected: number;
+  assignments_expired: number;
```

Comment:
We have an interceptor inside API to convert camelCase objects to snake_case so let's use camelCase inside the code. Also, I think this should be inside `stats.dto.ts`